spec/factories/models/pages.rb

```diff
@@ -1,7 +1,17 @@
+# When an OpenStruct is converted to json, it inludes @table.
+# We inherit and overide as_json here to use to contain answer_settings, which
+# is a json hash converted into an object by ActiveResource. Using a plain hash
+# for answer_settings means there is no .access to attributes.
+class DataStruct < OpenStruct
+  def as_json(*args)
+    super.as_json["table"]
+  end
+end
```

Comment:
Is there a better way to do this? The answer_settings doesn't have a model of their own and so can't be created using a dedicated factory. The answer_settings themselves need to be accessed using dot notation so plain old hashes are out. 

Using OpenStruct on it's own doesn't work either as it can be serialised to json for the session tests.